subprojects/snapshots/src/test/groovy/org/gradle/internal/fingerprint/impl/FingerprintCompareStrategyTest.groovy

```diff
@@ -103,29 +96,23 @@ class FingerprintCompareStrategyTest extends Specification {
         expect:
         changes(strategy, includeAdded,
             [:],
-            ["one-old": fingerprint("one")]
-        ) as List == [removed("one-old": "one")]
+            ["old/one": fingerprint("one")]
+        ) as List == [removed("old/one": "one")]
 
         where:
         strategy   | includeAdded
         NORMALIZED | true
-        NORMALIZED | false
         ABSOLUTE   | true
         ABSOLUTE   | false
     }
 
     @Unroll
-    def "non-trivial removal (#strategy, include added: #includeAdded)"() {
+    def "non-trivial removal"() {
```

Comment:
The strategy (`NORMALIZED`) should still be mentioned in the test title. The `@Unroll` annotation is now unnecessary. Please check the other cases, too.

Reply:
I've moved the `@Unroll` to the class in the other PR and added the strategy to every test method name (wasn't always there [before either](https://github.com/gradle/gradle/pull/9407/files#diff-5ca58042d95834e874cea1428f997913L160)).
Can we please merge this in?

Reply:
<img width="923" alt="Screenshot 2019-05-16 at 18 26 03" src="https://user-images.githubusercontent.com/1841944/57866586-438e9500-7808-11e9-8572-28100a3ef597.png">